build.gradle

```diff
@@ -0,0 +1,248 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+// TODO TEST switch to Kotlin DSL which is now the default and apparently better
+// TODO TEST avoid early configuration, see
+//           https://docs.gradle.org/current/userguide/task_configuration_avoidance.html
+//           need to avoid withType as well apparently?
+
+plugins {
+	id 'java'
+	id 'war'
+	id 'jacoco'
+	id 'org.ajoberstar.grgit' version '4.1.1'
+	id 'com.github.johnrengelman.shadow' version '8.1.1'
```

Comment:
what are these two for?

Reply:
First one will be for pulling the git commit for building into the server, so we don't have to rely on kb-sdk compile. Second one will be for building a shadow jar, which Gradle can't do out of the box

Reply:
They're both used in auth2 if you want to see examples